dev/TwoPaneView/TwoPaneView.h

```diff
@@ -64,4 +65,6 @@ class TwoPaneView :
     tracker_ref<winrt::RowDefinition> m_rowTop{ this };
     tracker_ref<winrt::RowDefinition> m_rowMiddle{ this };
     tracker_ref<winrt::RowDefinition> m_rowBottom{ this };
+    winrt::event_token m_windowSizeChangedToken{};
```

Comment:
can you please use a revoker (similar to the ones above for loaded ? they get automatically collected, so you don't need a destructor in that case. The unregistration will happen when the object is destroyed automatically.

Reply:
We can't use a tracker_ref, event_token is a value type, not a reference type.

Reply:
A revoker is different, it looks like you want winrt::Window::SizeChanged_revoker. You can check other revokers in the project for the usual pattern.

Reply:
Yes! I will update the PR this afternoon